### PR TITLE
docs: update template

### PR DESCRIPTION
**Pull request by violenil**
Part of https://github.com/jina-ai/finetuner/issues/451
Closes https://github.com/jina-ai/finetuner/issues/452 
 
Updating the template for writing guides.

- [ ]  This PR references an open issue
- [ ] I have added a line about this change to CHANGELOG

### CODE REVIEW

File: docs/_templates/template_ft_in_action.md [hunk 1]
```diff
@@ -6,27 +6,28 @@ This is a template for the documentation guides of Finetuner in action, with the
...
 
-## Choosing the model
+## Backbone model
 Mention which model will be used in your fine-tuning task. Feel free to add a `See Also` {admonition} for supplementary info on the model, perhaps a relevant paper or site.
 
 You can also add a `Tip` {admonition} for how the user can view all available models, also referring to the `Choose backbone model` documentation.
 
 
-## Creating a fine-tuning run
+## Fine-tuning
```
Comment (jupyterjazz) [inline docs/_templates/template_ft_in_action.md:28]: One important thing is that the code, when combined, should be runnable. So we should add a subsection with finetuner.login() inside `Fine-tuning` showing how to login to jina ecosystem.

**Review by jupyterjazz — COMMENTED** [review]
Looks good! 
I'd say let's leave this PR open and when we start working on the guides we'll have a better idea whether the template needs to be modified or not. 

please also update the changelog.

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: docs/_templates/template_ft_in_action.md (hunk 1 above)
Comment (violenil) [reply docs/_templates/template_ft_in_action.md:28]: Cant this `finetuner.login()` just be added to your code example in the fine-tuning section and then add a warning that users must login? There is a whole section `Login to Finetuner` in the `Step by step` so I don't think the guides need to have a separate section for this.

**Pull request closed without merging**